<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background: rgb(46, 47, 48);
	color: white;
}

QToolTip
{
	background-color: black;
	color: red;
	
	/*font: 8pt &quot;Segoe UI&quot;;*/
}

QMenuBar
{
	/*font: 11pt &quot;Segoe UI&quot;;*/
	color: rgb(194, 194, 194);;
	spacing: 5px;
	background: black;
}
QMenuBar::item:selected
{
	background: rgb(111, 111, 111); 
}
QMenuBar::item:pressed 
{
    background: red;
}

QMenu
{
	background-color: rgb(47, 47, 47);
	margin: 4px;
	icon-size: 12px;
}
QMenu::separator 
{
    height: 1px;
    background: red;
    margin: 2px;
}
QMenu::item:selected
{
	background-color: rgb(0, 66, 99);
}

QToolBar
{
	background-color: black;
	spacing: 18px;
}

QToolButton
{
	background-color: transparent;
	border: none;
	border-bottom: 1px solid red;
}
QToolButton::pressed
{
	border: none;
	border-bottom: 1px solid white;
}

QScrollBar:vertical
{
	width: 13px;
}
QScrollBar::down-button
{
	background: rgb(50, 50, 50);
	border: 4px solid red;
}
QScrollBar::down-arrow::vertical, QScrollBar::up-arrow:vertical
{
	border-image: url(:/tree_branch_open);
}
QScrollBar::handle
{
	border: 1px solid gray;
	border-radius: 4px;
	background: rgb(79, 80, 80);
}
QScrollBar::sub-page, QScrollBar::add-page
{
	background: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="Body" native="true">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit
{
	color: white;
	background: black;

	selection-color: white;
	selection-background-color: rgb(0, 66, 99);

	border: 1px solid rgb(98, 98, 98);	
	padding: 5px;
}
QPlainTextEdit::focus
{
	border: 1px solid #003333;
}

#folder_tree_container
{
	background-color: rgb(34, 34, 34);
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="folder_tree_container" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QTreeView
{
	background: #101314;
}
QTreeView::branch:has-siblings:!adjoins-item {
    border-image: url(:/icons/tree_vline.png) 0;
}

QTreeView::branch:has-siblings:adjoins-item {
    /*border-image: url(:/icons/tree_branch_open.png) 0;*/
}

QTreeView::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/icons/tree_branch_end.png) 0;
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/icons/tree_branch_closed.png);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/icons/tree_branch_open.png);
}

</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="treeView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CodeEditor" name="textEditor">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="plainText">
          <string>fasdasdas</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1014</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>101</y>
      <width>138</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Folder"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPaste"/>
    <addaction name="actionSelect_All"/>
    <addaction name="actionCopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/new_file.png</normaloff>:/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>casdasds</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/open_file.png</normaloff>:/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste             Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All      Ctrl+A</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy             Ctrl+C</string>
   </property>
  </action>
  <action name="actionOpen_Folder">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/icons/open_folder.png</normaloff>:/icons/open_folder.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
